### DISCUSSION TITLE
this.third.physics is undefined

### DISCUSSION THREAD

**Discussion by tautologias345** [enable3d/enable3d#289] [General]
I'm using Enable3D as 3D Extension for Phaser. Why it is undefined?

**Comment by vitalspace** [comment]
Unfortunately we are not diviners. Can you post the error and code please?

**Comment by tautologias345** [comment]
![image](https://user-images.githubusercontent.com/37247191/170518637-96a1200a-4ad2-4d8f-9f59-31c412933eda.png)

![image](https://user-images.githubusercontent.com/37247191/170518736-d751910a-6ab9-43fe-8468-2319e009d50d.png)

**Comment by yandeu** [comment]
It's `this.third.physics.add.existing()`

**Reply by tautologias345** [reply]
![image](https://user-images.githubusercontent.com/37247191/170528014-ce8466e1-3b3b-4168-bafe-9149a5c661c6.png)

![image](https://user-images.githubusercontent.com/37247191/170528097-599fc205-76b6-42fc-b648-f98a93645f0c.png)

**Comment by tautologias345** [comment]
```
var config = {
  type: Phaser.AUTO,
  transparent: true,
  width: 1280,
  height: 720,
  scale: {
	  mode: Phaser.Scale.ScaleModes.FIT,
	  autoCenter: Phaser.Scale.Center.CENTER_BOTH,
	  zoom: Phaser.Scale.Zoom.MAX_ZOOM
  },
  scene: [LabirintoInclinador],
  input: {
	  gamepad: true
  }
};
  
var game = new Phaser.Game(config);
  
window.addEventListener('load', function(){
	enable3d(game).withPhysics('https://www.eduardodequadros.site/infoprodutos/lib/ammo.js/builds');
});
```

**Reply by yandeu** [reply]
Your setup is wrong. Try:

```js
window.addEventListener('load', () => {
  enable3d(() => new Phaser.Game(config)).withPhysics('https://www.eduardodequadros.site/infoprodutos/lib/ammo.js/builds')
})
```
